R/PostDefinition.R

```diff
@@ -101,7 +101,12 @@ postDefinition <- function(baseUrl, name, category, definition) {
                     .data$isExcluded,
                     .data$includeMapped,
                     .data$includeDescendants)
-    expression <- RJSONIO::toJSON(x = items, digits = 23, pretty = TRUE)
+
+    itemsTranspose <- apply(items, 1, function(item) {
+      list(item)
+    })
+
+    expression <- RJSONIO::toJSON(x = itemsTranspose, digits = 23, pretty = TRUE)
```

Comment:
I'd revise this to use the .toJSON() function that is private to this package as detailed [here](https://github.com/OHDSI/ROhdsiWebApi/blob/master/R/Private.R#L177).  This way the magic "23" is not floating around the code base.